DP3TApp/Screens/Vaccination/NSVaccinationInfoViewController.swift

```diff
@@ -90,36 +92,60 @@ private class VaccinationInfoContentView: NSSimpleModuleBaseView {
     // MARK: - Setup
 
     private func setup() {
+        let bookNowLabel = NSLabel(.textBold)
+        bookNowLabel.text = "vaccination_impf_check_title".ub_localized
+
+        contentView.addSpacerView(NSPadding.large + NSPadding.small)
+        contentView.addArrangedView(bookNowLabel, insets: .zero)
+
+        let bookNowTextLabel = NSLabel(.textLight)
+        bookNowTextLabel.text = "vaccination_impf_check_info_text".ub_localized
+
+        contentView.addSpacerView(NSPadding.medium + 2.0)
+        contentView.addArrangedView(bookNowTextLabel, insets: .zero)
+
+        let button = NSButton(title: config?.vaccinationBookingInfo.value?.impfcheckButton ?? "vaccination_impf_check_action".ub_localized, style: .normal(.ns_blue))
```

Comment:
On Android we hide the complete view if the config values are null, this gives us the option to completely remove the view via config. Defaulting the to packed strings is therefore less elegant.